File: examples/csharp/cone.cs
using Kitware.VTK;

class Cone
{
    static void Main()
    {
        var cone = vtkConeSource.New();
        cone.SetResolution(8);

        var mapper = vtkPolyDataMapper.New();
        mapper.SetInputConnection(cone.GetOutputPort());

        var actor = vtkActor.New();
        actor.SetMapper(mapper);

        var renderer = vtkRenderer.New();
        renderer.AddActor(actor);
        renderer.SetBackground(0.2, 0.3, 0.4);

        var window = vtkRenderWindow.New();
        window.AddRenderer(renderer);
        window.SetSize(640, 480);

        var interactor = vtkRenderWindowInteractor.New();
        interactor.SetRenderWindow(window);

        window.Render();
        interactor.Start();
    }
}

File: examples/javascript/cone.js
const cone = vtk.vtkConeSource();
await cone.setResolution(8);

const mapper = vtk.vtkPolyDataMapper();
await mapper.setInputConnection(await cone.getOutputPort());

const actor = vtk.vtkActor();
await actor.setMapper(mapper);

const renderer = vtk.vtkRenderer();
await renderer.addActor(actor);
await renderer.setBackground(0.2, 0.3, 0.4);
await renderer.resetCamera();

const canvasSelector = "#vtk-wasm-window";
const renderWindow = vtk.vtkRenderWindow({ canvasSelector });
await renderWindow.addRenderer(renderer);

const interactor = vtk.vtkRenderWindowInteractor({
  canvasSelector,
  renderWindow,
});
await interactor.interactorStyle.setCurrentStyleToTrackballCamera();

await interactor.start();
